package.json

```
@@ -173,7 +173,7 @@
     "stylelint-config-twbs-bootstrap": "^15.1.0",
     "tarteaucitronjs": "^1.19.0",
     "terser": "^5.36.0",
-    "vnu-jar": "^23.4.11"
+    "vnu-jar": "23.4.11"
```

The version doesn't match the one on Bootstrap :
```suggestion
    "vnu-jar": "24.10.17"
```
Defined in [this commit](https://github.com/twbs/bootstrap/commit/c2a7d686de56f5cdb829c63e1b3849a2c04409d0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R157)

---

Oh I just saw that you got back from this version in a previous commit. I suppose it's linked to the compatibility issue with java version in the runner if it's related to [this error message](https://app.netlify.com/sites/boosted/deploys/674de27324fa040008dc6958#L289)

I'm a netlify newbie but from the documentation it seems we can't do anything about that on our side :/